modules/sbt-dependency-manager/build.sbt

```diff
@@ -36,6 +36,11 @@ libraryDependencies <++= scalaBinaryVersion {
   case "2.11" => ("com.ning" % "async-http-client" % "[1.6.5, 1.6.5]" force() exclude("org.jboss.netty", "netty"))::Nil
 }
 
+libraryDependencies ++= Seq(
+  "io.get-coursier" %% "coursier" % "1.0.0-RC3",
```

Comment:
crap. though, I've seen this in tests only.
so if it's only `specs2` uses scalaz, solution is simple. 🗑  specs2, in favour of `scalaTest`.

```
[info]   |   +-org.specs2:specs2-core_2.11:2.3.12 [S]
[info]   |     +-org.specs2:specs2-matcher_2.11:2.3.12 [S]
[info]   |       +-org.specs2:specs2-common_2.11:2.3.12 [S]
[info]   |         +-org.scala-lang:scala-reflect:2.11.8 [S]
[info]   |         +-org.scalaz:scalaz-concurrent_2.11:7.0.6 (evicted by: 7.2.8)
[info]   |         +-org.scalaz:scalaz-concurrent_2.11:7.2.8 [S]
[info]   |         | +-org.scalaz:scalaz-core_2.11:7.2.8 [S]
[info]   |         | +-org.scalaz:scalaz-effect_2.11:7.2.8 [S]
[info]   |         |   +-org.scalaz:scalaz-core_2.11:7.2.8 [S]
[info]   |         |
[info]   |         +-org.scalaz:scalaz-core_2.11:7.0.6 (evicted by: 7.2.8)
[info]   |         +-org.scalaz:scalaz-core_2.11:7.2.8 [S]
```